src/H5FDsplitter.h

```diff
@@ -93,14 +60,46 @@ H5_DLL hid_t H5FD_splitter_init(void);
 /**
  * \ingroup FAPL
  *
- * \todo Add missing documentation
+ * \brief Sets the file access property list to use the splitter driver
+ *
+ * \fapl_id
+ * \param[in] config_ptr Configuration options for the VFD
+ * \returns \herr_t
+ *
+ * \details H5Pset_fapl_splitter() sets the file access property list identifier,
+ *          \p fapl_id, to use the splitter driver.
+ *
+ *          The splitter VFD echoes file manipulation (e.g. create, truncate)
+ *          and write calls to a second, write-only file.
+ *
+ *          \note The splitter VFD should not be confused with the split VFD,
+ *          which is a simplification of the multi VFD and creates separate
+ *          files for metadata and data.
+ *
+ * \since 1.10.7, 1.12.1
  */
 H5_DLL herr_t H5Pset_fapl_splitter(hid_t fapl_id, H5FD_splitter_vfd_config_t *config_ptr);
 
 /**
  * \ingroup FAPL
  *
- * \todo Add missing documentation
+ * \brief Gets splitter driver properties from the the file access property list
```

Comment:
double "the"

Reply:
Oops. Will fix in a subsequent PR (there will be a couple more VFD PRs).